pkg/hostagent/config.go

```diff
@@ -307,4 +310,5 @@ func (config *HostAgentConfig) InitFlags() {
 	flag.BoolVar(&config.EnableNodePodIF, "enable-nodepodif", false, "Enable NodePodIF")
 	flag.StringVar(&config.EPRegistry, "ep-registry", "", "Enable PodIF")
 	flag.BoolVar(&config.OvsHardwareOffload, "enable-sriov-config", false, "SRIOV config and ovs hardware offload feature")
+	flag.StringVar(&config.DpuOvsDBSocket, "dpu-ovsdb-socket", "tcp:192.168.200.2:6640", "TCP socket on DPU to connect to")
```

Comment:
Is it okay to default to this specific address? How did we choose this one?

Reply:
Sumit, those are part of my changes. yeah it should be ok because this is the private network between DPU and host and it needs to be setup before the host can talk to the DPU and since the customer will not have access to the DPU there is no way for him to set this up via any config we generate. This code review is missing changes on DPU itself so its not clear without that context.